node-red-node-wot/VERSIONING.md

```diff
@@ -0,0 +1,10 @@
+# Versioning
+
+The package uses [semantic versioning](https://semver.org/) with the following meanings to different versions:
+
+- Patch: Under the hood, documentation and UI annotation fixes:
+  - The user flow sees no change other than some human-readable text within Node-RED.
+  - Even if node-wot gets a major overhaul but nothing changes for the Node-RED user, this versioning applies. For example, changing to .value() function was a major change in node-wot but this would have no impact to the Node RED user.
```

Comment:
For example, if the node-wot used in the Node-RED flow on the server is upgraded to a version that requires .value(), would the client developer need to change the client implementation?
If the developer of the server and the developer of the client are different, the developer of the server created in Node-RED may need to notify the developer of the client, but if only the Patch version has changed, the developer of the server cannot determine if the client is affected. If only the Patch version has changed, it would be difficult for the server developer to determine if the client is affected or not.

Reply:
In the specific example of "For example, if the node-wot used in the Node-RED flow on the server is upgraded to a version that requires .value(), would the client developer need to change the client implementation?", there should be no change required.

In general, the server and client should never need to know the node-wot or node-red packaged version of each other. TD should be the only source of truth.

Reply:
Thank you for your response.

I understand that even if the server node-wot is a later version that supports .value(), the client node-wot can communicate with the pre .value() version.

I think the versioning policy you have created is fine.